spec/helpers/errors_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
do we need this file?

Reply:
read first how to write specs or try by your own and see if needed or not

Reply:
app/helpers/errors_helper.rb is an empty file.

Reply:
fixed, no we don't need that empty file